data_aquisition_layer/.vscode/arduino.json

```diff
@@ -0,0 +1,6 @@
+{
+    "sketch": "data_aquisition_layer.ino",
+    "board": "espressif:esp32:esp32",
+    "configuration": "PartitionScheme=default,FlashMode=qio,FlashFreq=80,FlashSize=4M,UploadSpeed=921600,DebugLevel=none",
+    "port": "/dev/cu.SLAB_USBtoUART"
```

Comment:
We might need to add this file to the gitignore as well because the port can change from machine to machine.